packages/GA4Client/src/common.js

```diff
@@ -132,6 +132,15 @@ Common.prototype.standardizeParameters = function (parameters) {
 };
 
 Common.prototype.standardizeName = function (name) {
+    try {
+        name = window.GoogleAnalytics4Kit.setCustomNameStandardization(name);
+    } catch (e) {
+        console.warn(
```

Comment:
I think using `console.error` is more appropriate, as this is an error.

```suggestion
        console.error(
```